src/main/resources/resources/logisim/strings/proj/proj.properties

```diff
@@ -36,6 +36,8 @@ replaceExtensionTitle = Replace Extension
 templateOpenError = Could not open template: %s
 templateOpenErrorTitle = Error Loading Template
 UsedLibraryToolnameError = The filename may not be equal to a circuit name, tool name, or library name used in your project.\n The specified filename equals to one of these names.
+ProjUnableToCreate = Unable to create project, aborting...
+ProjExistsUnableToCreate = A project called %s already exists in the selected directory, please select another location.
```

Comment:
Can we have this rephrased as (IMHO) the message do not clearly link project name with the presence of the folder named after it, which is the real reason. Maybe something like `Folder named "%s" already exists in selected directory. Please select another location or use different project name.`

Reply:
Done